### DISCUSSION TITLE
Timer based rule after reboot

### DISCUSSION THREAD

**Discussion by lafriedrich** [arendst/Tasmota#19512] [Support request] [answered]
On Sonoff SV, I have 2 GPIO as inputs, and the following rule is enabled. Usually, things work fine, but if a switch activates Backlog Power1 1 and RuleTimer, then a watchdog/random reboot happens for some reason, all timers are set to zero and power remains on indefinitely.

Rule1
ON Switch5#state=2 DO Backlog Power1 1; RuleTimer1 240 ENDON
ON Switch4#state=2 DO Backlog Power1 1; RuleTimer1 1200 ENDON
ON Rules#Timer=1 DO Power1 off ENDON

Tried
PowerOnState 0 
SetOption0 0
SetOption63 1

Should be a silly question, but how can I ensure that power is off after any sort of reboot?

**Comment by sfromis** [comment]
`PowerOnState` has nothing to do with rebooting, it is only relevant after power on. Using `SetOption0` means that a save of settings (including power state) is not triggered by changing power state, but other things will still trigger this unless turning off `SaveData`. 

You could have a trigger like `on Power1#boot==1 do Power1 off endon` to quickly turn off power if on at boot time.

**Reply by lafriedrich** [reply, marked as answer]
thanks for pointing me on the right direction! 

```
on system#boot do Power1 off endon
```  
did the trick
